test/e2e/suite_test.go

```diff
@@ -119,6 +129,7 @@ var _ = BeforeSuite(func() {
 	Expect(configv1.AddToScheme(scheme.Scheme)).NotTo(HaveOccurred())
 	Expect(templatev1.AddToScheme(scheme.Scheme)).NotTo(HaveOccurred())
 	Expect(appsv1.AddToScheme(scheme.Scheme)).NotTo(HaveOccurred())
+	Expect(argoapi.AddToScheme(scheme.Scheme)).NotTo(HaveOccurred())
```

Comment:
```suggestion
```

This Expect is not required. Already added previous 2 `Expect` for `argov1alpha1api` & `argov1beta1api` will register the required CR.